releasenotes-dca/notes/add-exponential-backoff-to-syncinformer-9385d768f8ca21ce.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Add exponential backoff for syncing with the kubernetes informers. The retry period and <> can be configured with <>, <>.
```

Comment:
I meant to replace the `<>` depending on what settings we decide to expose :)

Reply:
Hahaha, I didn’t realise this before